streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -145,6 +146,10 @@ private[ui] class StreamingPage(parent: StreamingTab)
   private val listener = parent.listener
   private val startTime = System.currentTimeMillis()
 
+  parent.parent.attachHandler(
+    StreamingApiRootResource.getServletHandler(parent.parent, listener, startTime)
```

Comment:
I know its small but I'd rather not commit code with the plan to change it later.